src/output.rs

```diff
@@ -54,6 +54,43 @@ fn print_trailing_slash<W: Write>(
     Ok(())
 }
 
+// Trying to copy: https://www.gnu.org/software/coreutils/quotes.html
+fn path_needs_quoting(path: &str) -> i8 {
+    // If it contains any special chars we return single quote
+    if path.contains(" ") || path.contains("$") || path.contains("\"") {
```

Comment:
This is probably not careful enough.  See https://github.com/tavianator/bfs/blob/b8ed989642b9f0f6c1301bcff6f1498935cbd81c/src/bfstd.c#L906-L918 for example